Content.Shared/Access/Systems/AccessReaderSystem.cs

```diff
@@ -115,9 +122,24 @@ public bool IsAllowed(EntityUid entity, AccessReaderComponent reader)
         /// </summary>
         /// <param name="accessTags">A list of access tags</param>
         /// <param name="reader">An access reader to check against</param>
-        public bool AreAccessTagsAllowed(ICollection<string> accessTags, AccessReaderComponent reader)
+        /// <param name="logAccess">Should successful access be logged</param>
+        public bool AreAccessTagsAllowed(IEnumerable<string> accessTags, AccessReaderComponent reader, bool logAccess = true)
         {
-            if (reader.DenyTags.Overlaps(accessTags))
+            var providedTags = new ProvidedAccessList();
+            providedTags.AddAccessTags("", accessTags, !logAccess);
```

Comment:
I havent done a dive in the changes here but i will just comment that it is sus feeding an empty string like this to a method